src/components/NetworkSwitcher.tsx

```diff
@@ -0,0 +1,45 @@
+import { tenderlyBase } from '@/data/wagmi/config/config.default';
+import { MainnetNetwork, BaseNetwork } from '@/modules/icons';
+import { useChainModal } from '@rainbow-me/rainbowkit';
+import { base } from 'viem/chains';
+import { useChainId } from 'wagmi';
+import { Tooltip, TooltipArrow, TooltipContent, TooltipPortal, TooltipTrigger } from './ui/tooltip';
+import { Text } from '@/modules/layout/components/Typography';
+import { Trans } from '@lingui/macro';
+import { useConfigContext } from '@/modules/config/hooks/useConfigContext';
+import { Intent } from '@/lib/enums';
+
+export function NetworkSwitcer() {
```

Comment:
sorry to nitpick but we should probably fix this typo 😅

Reply:
New keyboard messing around with variable names 😅 I fixed this in the other network PR so I'll merge this one